recipes/r-metamix/meta.yaml

```diff
@@ -0,0 +1,108 @@
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  skip: true  # [win]
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-matrix
+    - r-rmpi
+    - r-data.table >=1.9.2
+    - r-ggplot2
+    - r-gtools
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+    - r-rmpi
+    - r-data.table >=1.9.2
+    - r-ggplot2
+    - r-gtools
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('metaMix')"           # [not win]
+    - "\"%R%\" -e \"library('metaMix')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://CRAN.R-project.org/package=metaMix
+  license: GPL-3
+  summary: Resolves complex metagenomic mixtures by analysing deep sequencing data, using a mixture
+    model based approach. The use of parallel Monte Carlo Markov chains for the exploration
+    of the species space enables the identification of the set of species more likely
+    to contribute to the mixture.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+# The original CRAN metadata for this package was:
+
+# Package: metaMix
+# Title: Bayesian Mixture Analysis for Metagenomic Community Profiling
+# Version: 0.3
+# Author: Sofia Morfopoulou <sofia.morfopoulou.10@ucl.ac.uk>
+# Maintainer: Sofia Morfopoulou <sofia.morfopoulou.10@ucl.ac.uk>
+# Depends: R (>= 3.2)
+# Imports: data.table (>= 1.9.2), Matrix, gtools, Rmpi, ggplot2
+# Suggests: knitr
+# VignetteBuilder: knitr
+# Description: Resolves complex metagenomic mixtures by analysing deep sequencing data, using a mixture model based approach. The use of parallel Monte Carlo Markov chains for the exploration of the species space enables the identification of the set of species more likely to contribute to the mixture.
+# License: GPL-3
+# LazyData: true
+# SystemRequirements: Open MPI (>=1.4.3)
+# RoxygenNote: 6.1.1
+# Encoding: UTF-8
+# NeedsCompilation: yes
+# Packaged: 2019-02-07 15:41:40 UTC; sophia
+# Repository: CRAN
+# Date/Publication: 2019-02-11 16:20:03 UTC
+
+# See
+# https://docs.conda.io/projects/conda-build for
+# more information about meta.yaml
+
+extra:
+  recipe-maintainers:
+    - sarah-buddle
```

Comment:
We can keep this CRAN metadata for refernce, but move it to the bottom. Also, the R team should be maintainers:

```suggestion
extra:
  recipe-maintainers:
    - conda-forge/r
    - sarah-buddle

# Package: metaMix
# Title: Bayesian Mixture Analysis for Metagenomic Community Profiling
# Version: 0.3
# Author: Sofia Morfopoulou <sofia.morfopoulou.10@ucl.ac.uk>
# Maintainer: Sofia Morfopoulou <sofia.morfopoulou.10@ucl.ac.uk>
# Depends: R (>= 3.2)
# Imports: data.table (>= 1.9.2), Matrix, gtools, Rmpi, ggplot2
# Suggests: knitr
# VignetteBuilder: knitr
# Description: Resolves complex metagenomic mixtures by analysing deep sequencing data, using a mixture model based approach. The use of parallel Monte Carlo Markov chains for the exploration of the species space enables the identification of the set of species more likely to contribute to the mixture.
# License: GPL-3
# LazyData: true
# SystemRequirements: Open MPI (>=1.4.3)
# RoxygenNote: 6.1.1
# Encoding: UTF-8
# NeedsCompilation: yes
# Packaged: 2019-02-07 15:41:40 UTC; sophia
# Repository: CRAN
# Date/Publication: 2019-02-11 16:20:03 UTC

```